techlibs/quicklogic/synth_quicklogic.cc

```diff
@@ -47,6 +47,9 @@ struct SynthQuickLogicPass : public ScriptPass {
         log("        - ap3: ArcticPro 3 \n");
         log("        - qlf_k4n8: qlf_k4n8 \n");
         log("\n");
+        log("    -no_ff_map\n");
+        log("        Specifying this switch turns off ff techmap.\n");
```

Comment:
I think we should also mention here 'By default ff tech map is enabled. Specifying this switch turns off ff tech map.'